scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/TaskLauncherInitializer.java

```diff
@@ -82,6 +81,8 @@
     /** iteration index: task was replicated sequentially */
     private int iterationIndex = 0;
 
+    private Map<String, String> genericInformation;
```

Comment:
yes they are unique and stored in the server in a HashMap. We can have a definition at job level and at task level for the same key. In that case, the task definition overrides the job definition